LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [2023] [Glauber Bannwart & Javier]
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍